### PR TITLE
Update XR SDK docs with new XR Management version and updated images

### PR DESCRIPTION
**Pull request by keveleigh**
## Overview

Updates our XR SDK docs with the new in-Unity flow due to an updated XR Management version. The older XR Management version appears to have been unpublished, so this is the only flow going forward.

Also added images to try to add clarity to where and how each legacy MRTK service should be updated to use the XR SDK versions.

## Changes
- Fixes: https://github.com/microsoft/MixedRealityToolkit-Unity/issues/7538

### CODE REVIEW

File: Documentation/GettingStartedWithMRTKAndXRSDK.md [hunk 1]
```diff
@@ -13,27 +13,64 @@ For MRTK 2.3, Windows Mixed Reality is supported on XR SDK.
...
-1. Click on the XR Plugin Management tab in the Project Settings window.
-1. Go to the Universal Windows Platform settings, then click the + to add a Windows MR Loader.
-1. Ensure that Initialize on Startup is checked.
-1. (**_Optional_**) Click on the Windows Mixed Reality tab under XR Plugin Management and create a custom settings profile to change the defaults.
+1. Click on the XR Plug-in Management tab in the Project Settings window.
+1. Go to the Universal Windows Platform settings and ensure Windows Mixed Reality is checked under Plug-in Providers.
+1. Ensure that Initialize XR on Startup is checked.
+1. (**_Optional_**) Click on the Windows Mixed Reality tab under XR Plug-in Management and create a custom settings profile to change the defaults. If the list of settings are already there, no profile needs to be created.
 
 ![Plugin management](Images/XRSDK/PluginManagement.png)
```
Comment (CDiaz-MS) [inline Documentation/GettingStartedWithMRTKAndXRSDK.md:26]: It looks like the new screenshots were taken with the unity dark theme.  I think we decided that we would use the light theme for the docs moving forward.

Comment (keveleigh) [reply Documentation/GettingStartedWithMRTKAndXRSDK.md:26]: Oof yeah, good catch! I'll redo

Comment (keveleigh) [reply Documentation/GettingStartedWithMRTKAndXRSDK.md:26]: Updated

File: Documentation/GettingStartedWithMRTKAndXRSDK.md [hunk 2]
```diff
@@ -13,27 +13,64 @@ For MRTK 2.3, Windows Mixed Reality is supported on XR SDK.
 ### Required in Unity
 
 1. Go into Unity's Package Manager and install the Windows XR Plugin package, which adds support for Windows Mixed Reality on XR SDK. This will pull down a few dependency packages as well. Ensure the following all successfully installed:
-   1. XR Management
+   1. XR Plugin Management
```
Comment (davidkline-ms) [inline Documentation/GettingStartedWithMRTKAndXRSDK.md:16]:
```suggestion
   1. XR Plug-in Management
```

Comment (keveleigh) [reply Documentation/GettingStartedWithMRTKAndXRSDK.md:16]: I (unfortunately) intentionally left these mismatched, as they match the spellings in Unity:

image

image

**Pull request merged**